rasa/nlu/utils/spacy_utils.py

```diff
@@ -49,11 +48,9 @@ def load_model(spacy_model_name: Text) -> "Language":
             return spacy.load(spacy_model_name, disable=["parser"])
         except OSError:
             raise InvalidModelError(
-                "Model '{}' is not a linked spaCy model.  "
-                "Please download and/or link a spaCy model, "
-                "e.g. by running:\npython -m spacy download "
-                "en_core_web_md\npython -m spacy link "
-                "en_core_web_md en".format(spacy_model_name)
+                "Please confirm that {} is an available spaCy model. "
+                "You may need to download one upfront. For example:\npython -m spacy download "
+                "en_core_web_md".format(spacy_model_name)
             )
 
```

Comment:
Some extra changes should be made to ensure that a user passes a model name. That means removing lines like [this](https://github.com/RasaHQ/rasa/pull/7869/files#diff-cb47a94f8eba5f3264f3da2701ac4d56dec845bca6a1d92915b7785a9e76be8fR72-R74) and replacing them with a checking mechanism that throws appropriate errors.